src/gettingstarted/own-code/project-configuration.md

```diff
@@ -0,0 +1,87 @@
+# Import your own code
+
+## Configuring projects
+
+In the previous step, you created a new project on Platform.sh using the CLI. Now, there are a few configuration steps left that will help Platform.sh know what to do with your application during builds and deployments.
+
+1. **Consult a template alongside this guide**
+
+    As you go through this guide, example files will be provided that will give you a good impression of how to configure applications on Platform.sh in the language of your choice. However, since they are simple examples and your own application may require more detailed configuration than those examples address, it is recommended that you take a look at our maintained templates for additional guidance.
+
+    Select a language and choose one or more templates that most closely resemble your application and keep the template in another tab as you continue through this guide. Using these two resources together is the fastest way to correctly configure your project for Platform.sh.
+
+    <div class="column">
+      <div id="php"></div>
+      <div id="python"></div>
+    </div>
+    <div class="column">
+      <div id="nodejs"></div>    
+      <div id="ruby"></div>
+    </div>
+    <div class="column">
+      <div id="go"></div>
+      <div id="java"></div>
+    </div>
+
+    <script>
+        var descPathPHP = getPathObj("/languages/php.html#project-templates", "PHP");
+        var descButtonPHP = {type: "basicFull", path: descPathPHP, div: "php"};
+        makeButton(descButtonPHP);
+
+        var descPathPython = getPathObj("/languages/python.html#project-templates", "Python");
+        var descButtonPython = {type: "basicFull", path: descPathPython, div: "python"};
+        makeButton(descButtonPython);
+
+        var descPathNode = getPathObj("/languages/nodejs.html#project-templates", "Node.js");
+        var descButtonNode = {type: "basicFull", path: descPathNode, div: "nodejs"};
+        makeButton(descButtonNode);
+
+        var descPathRuby = getPathObj("/languages/ruby.html#project-templates", "Ruby");
+        var descButtonRuby = {type: "basicFull", path: descPathRuby, div: "ruby"};
+        makeButton(descButtonRuby);
+
+        var descPathGo = getPathObj("/languages/go.html#project-templates", "Go");
+        var descButtonGo = {type: "basicFull", path: descPathGo, div: "go"};
+        makeButton(descButtonGo);
+
+        var descPathJava = getPathObj("/languages/java.html#project-templates", "Java");
+        var descButtonJava = {type: "basicFull", path: descPathJava, div: "java"};
+        makeButton(descButtonJava);
+    </script>
```

Comment:
There's no reason for these buttons to be Javascript.  They should just be basic A elements in an ordered list, with classes for styling.  As is, this isn't accessible or SEO-able.

Reply:
We should also look into making the list of supported templates automated if we can.  Ideally at compile time, not runtime.